test/config_parser_SUITE.erl

```diff
@@ -1794,22 +1794,18 @@ mod_jingle_sip(_Config) ->
       <<"listen_port">> => 5601,
       <<"local_host">> => <<"localhost">>,
       <<"sdp_origin">> => <<"127.0.0.1">>
-     },
+    },
```

Comment:
As no options are required, I think we need to test them one-by-one, not grouped as `Base`.
This rework can be done as a part of this PR or separately, the decision is up to you.